crates/uv/src/commands/python/install.rs

```diff
@@ -302,114 +312,136 @@ pub(crate) async fn install(
             .expect("We should have a bin directory with preview enabled")
             .as_path();
 
-        let target = bin.join(installation.key().versioned_executable_name());
+        let targets = if (default || is_default_install)
```

Comment:
Why `is_default_install` here? I thought that meant something distinct from `default`.

Reply:
> If you run `uv python install` (without arguments), we include the --default flag implicitly to populate python and python3 for the "default" install version.

When you run` uv python install` without arguments, it's a "default install". The idea is for that bootstrapping experience to be complete.

Reply:
(If you've previously installed Python with uv, using `uv python install` without arguments has no effect)

Reply:
I see, ok. But if there's a `.python-version` file, it does _not_ count as a "default" install.

Reply:
Yeah, I'm not really sure if it makes sense but I would find it surprising for my "default" version to be set when installing the Python versions needed for a project, e.g., I noticed this for uv.